### PR TITLE
graphql: refactor and prefer static GraphQL where possible

### PR DESCRIPTION
**Pull request by dotansimha**
Closes https://github.com/graphprotocol/graph-node/issues/3203 

- [x] Move static GraphQL to be in `meta.graphql`
- [x] Added missing `@skip` and `@include` direcrives to the type definitions. 
- [x] Added descriptions to all static type defs.

### CODE REVIEW

**Comment by dotansimha** [comment]
@lutter rebased this one as well, should be quick and easy review ;)

File: graphql/src/schema/meta.graphql
Comment (lutter) [inline graphql/src/schema/meta.graphql:5]: We don't have a `Float` type - not sure what will happen if somebody tries to use that, but I'd suspect that it will blow up in some way.

**Review by lutter — APPROVED** [review]
Just remove the `Float` type, and this is good to go.

File: graphql/src/schema/meta.graphql
Comment (dotansimha) [reply graphql/src/schema/meta.graphql:5]: Thanks for the review @lutter ! 
We are using it as part of the introspection schema (https://github.com/graphprotocol/graph-node/blob/393d0dca8c25a184f140da7cc62596a26d4580e5/graphql/src/introspection/schema.rs#L17), and it's also a built-in in GraphQL, so are you sure it's a good direction to drop it?

Comment (dotansimha) [reply graphql/src/schema/meta.graphql]: I rebased and pushed a fix for it, but not merging yet, waiting for your decision on that. 
Thanks!

Comment (lutter) [reply graphql/src/schema/meta.graphql]: I just checked the code, and we actually do handle it - you are right, we should leave the `Float` in then.

Comment (dotansimha) [reply graphql/src/schema/meta.graphql:5]: Fixed, thanks! will merge when CI clears

**Pull request merged**
